js/components/dashboard/dropdown.js

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react'
+import { SORT_BY_NAME, SORT_BY_MOST_PROGRESS, SORT_BY_LEAST_PROGRESS } from '../../actions/dashboard'
+
+export default class Dropdown extends PureComponent {
```

Comment:
I called this `Dropdown` thinking that it could be made generic moving forward, even though it's only for sorting now. Do you think it's ok to leave like this?

Reply:
I'd rather call it SortByDropdown or something like this. Right now it isn't generic at all and this name can be a bit misleading. If we ever need to create a generic Dropdown, we'll just do it, I don't think it's a big deal.